File: app/src/main/java/ru/ulstu_team/ulstuschedule/ui/common/student/StudentScheduleMvpView.java
package ru.ulstu_team.ulstuschedule.ui.common.student;

import java.util.List;

import ru.ulstu_team.ulstuschedule.data.model.Lesson;
import ru.ulstu_team.ulstuschedule.ui.base.MvpView;

public interface StudentScheduleMvpView extends MvpView{

    void showSchedule(List<Lesson> lessons);

    void showEmptySchedule();

    void showError();

    void reload();

}

File: app/src/main/java/ru/ulstu_team/ulstuschedule/ui/common/student/StudentScheduleMvpView.kt
package ru.ulstu_team.ulstuschedule.ui.common.student

import ru.ulstu_team.ulstuschedule.data.model.Lesson
import ru.ulstu_team.ulstuschedule.ui.base.MvpView

interface StudentScheduleMvpView : MvpView {

    fun showSchedule(lessons: List<Lesson>)

    fun showEmptySchedule()

    fun showError()

}
